metagpt/actions/write_prd.py

```diff
@@ -132,8 +135,12 @@ async def _is_relative(self, new_requirement_doc, old_prd_doc) -> bool:
     async def _merge(self, new_requirement_doc, prd_doc, schema=CONFIG.prompt_schema) -> Document:
         if not CONFIG.project_name:
             CONFIG.project_name = Path(CONFIG.project_path).name
-        prompt = NEW_REQ_TEMPLATE.format(requirements=new_requirement_doc.content, old_prd=prd_doc.content)
-        node = await WRITE_PRD_NODE.fill(context=prompt, llm=self.llm, schema=schema)
+        prompt = REFINE_PRD_TEMPLATE.format(
```

Comment:
`REFINE_PRD_TEMPLATE` keep same with `REFINED_CODE_TEMPLATE` ? use `REFINED`, check all the context.